core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -9131,14 +9132,41 @@ public Sql schema(CalciteAssert.SchemaSpec schemaSpec) {
   }
 
   @Test public void testExtractEpoch() {
-    String query = "SELECT EXTRACT(EPOCH FROM DATE '2008-08-29')";
-    final String expectedBQ = "SELECT UNIX_SECONDS(DATE '2008-08-29')";
```

Comment:
Do not modify source query of previous test cases

Reply:
The existing test case expectedBQ is wrong and do not works on bq.
It gives:

No matching signature for function UNIX_SECONDS for argument types: DATE. Supported signature: UNIX_SECONDS(TIMESTAMP) at [4:8]

Reply:
reverted